src/python/pants/help/scope_info_iterator_test.py

```diff
@@ -51,7 +51,7 @@ def subsystem_dependencies(cls):
         scope_to_infos = dict((x.scope, x) for x in infos)
 
         it = ScopeInfoIterator(scope_to_infos)
-        actual = list(it.iterate([GLOBAL_SCOPE, "goal1", "goal2.task21", "goal3"]))
+        actual = list(it.iterate({GLOBAL_SCOPE, "goal1", "goal2.task21", "goal3"}))
```

Comment:
Why make this change? You're losing a guarantee of stable order. We know for a fact that there is no duplication with these elements, so a set wouldn't bring any benefit.

Reply:
`iterate` is explicitly sorting (alphabetically but with special treatment for Subsystems), so it makes sense to avoid imposing a specific order on the inputs to that method.